### ISSUE TITLE
struct crow::request has no member named get_body_params

### ISSUE THREAD

**Issue by FinalFantaser** [CrowCpp/Crow#709]
When I try to use the get_body_params method as specified in docs:
```c++
CROW_ROUTE(app, "/").methods(crow::HTTPMethod::POST)([](crow::request req){
        auto body{ req.get_body_params() };
        return crow::response(201);
    });
```

I encounter an error as follows:
`‘struct crow::request’ has no member named ‘get_body_params’|`

I used crow_all.h and latest v1.0+5 release downloaded from the Releases section, but neither of them has the get_body_params() reference.

**Comment by MichaelSB** [comment]
Docs apply to master. Sadly there was no release in quite some time now. Try:
`auto body = crow::query_string(req.body, false);`
as this is all get_body_params() does. Or use master instead.

**Comment by FinalFantaser** [comment]
> Docs apply to master. Sadly there was no release in quite some time now. Try: `auto body = crow::query_string(req.body, false);` as this is all get_body_params() does. Or use master instead. Edit: just realized this query_string constructor is not available in 1.0.x as well.

I compiled the master, no issue now. Thanks!
